test/runtests.jl

```diff
@@ -19,6 +19,7 @@ limit(a, N) = a == N + 1 ? 1 : a == 0 ? N : a
 
 if GROUP == "All" || GROUP == "Core"
       @testset begin
+        @safetestset "GSOC RootFinding solver" begin include("new_solver.jl") end
```

Comment:
This probably shouldn't be the first test since it's a higher level functionality

Also the name should just be Rootfinding Test